src/bucket.js

```diff
@@ -70,23 +69,26 @@ export default class Bucket {
    * @param  {Boolean} [options.safe]    The safe option.
    * @return {Collection}
    */
-  collection(name, options={}) {
-    return new Collection(this.client, this, name, this._bucketOptions(options));
+  collection(name, options = {}) {
+    return new Collection(
+      this.client,
+      this,
+      name,
+      this._bucketOptions(options)
+    );
   }
 
-
   /**
    * Retrieves bucket data.
    *
    * @param  {Object} [options={}]      The options object.
    * @param  {Object} [options.headers] The headers object option.
    * @return {Promise<Object, Error>}
    */
-  getData(options={}) {
-    const reqOptions = {...this._bucketOptions(options)};
-    const request = {...reqOptions, path: endpoint("bucket", this.name)};
-    return this.client.execute(request)
-      .then((res) => res.data);
+  getData(options = {}) {
+    const reqOptions = { ...this._bucketOptions(options) };
```

Comment:
Sadly, or actually fortunately, prettier is opinionated and doesn't expose configuration flags for that ;)